### DISCUSSION TITLE
[org] Evaluate each PR and each issue of the old repository

### DISCUSSION THREAD

**Discussion by artesim-zz** [Ortho4XP/Ortho4XP#19] [Ideas]
I already created two cards for this [in the Features kanban](https://github.com/orgs/Ortho4XP/projects/1).

And since we merged both the progressive_zl feature and @girotobial 's work on pinning
the dependencies, it's possible some issues (and maybe PRs) could be closed now.

**Comment by d41k4n** [comment]
Instead of re-creating the PR's in the dev branch (as suggested in the [kanban card](https://github.com/orgs/Ortho4XP/projects/1#card-71981055)) why not simply merge the open upstream PR's that are rather trivial (like new/updated provider definitions) and which can be automatically merged into the upstream master and then do a fetch into master? 

Just an idea since re-creating them into the dev branch seems like a lot of work...

**Reply by meznak** [reply]
This likely won't work as the repos diverge, but it would cut down on what needs to be redone here. It would require someone with write permissions on the original repo, and there's probably a reason that hasn't happened yet. I'm still getting familiar with the code, but would be happy to review some of them with a maintainer when I am.
